configmaptocrs/main.go

```diff
@@ -34,6 +34,7 @@ const (
 var (
 	resourcesNameSpace = "metallb-system"
 	inputDirPath       = "/var/input"
+	dataOnly           = flag.Bool("data-only", false, "set this to true if the source file includes only the configmap's data")
```

Comment:
"config as yaml" -> set this to true if the config is provided directly as yaml and not as the content of a configmap

Reply:
done